docs/multi-worker.md

```diff
@@ -0,0 +1,133 @@
+
+## Recommended implementation
+
+When enabling the multi-worker setup, it is recommended to ensure the following things:
+- Place the fluentds on separate Nodes
+- Increase the compute and memory resources 
+- Configure a buffer volume for fluentd
+- Do not use specific filter plugins
+  - detectExceptions [is not working](https://github.com/kube-logging/logging-operator/issues/1490)
+- Also scale [horizontally](https://kube-logging.dev/docs/logging-infrastructure/fluentd/#autoscaling)
+
+To ensure that the fluentd Pods have enough resources, a common approach is to use specific Nodes for the fluentds and to reserve enough computing and memory resources. A new nodePool should be created with a specific label and a taint. Ideally, the nodeType is compute-optimized. It could look like the following:
+```yaml
+apiVersion: v1
+kind: Node
+metadata:
+  labels:    
+    type: cpu
+  name: node1
+spec:
+  taints:
+  - effect: NoSchedule
+    key: type
+    value: cpu
+```
+
+The corresponding setting in the Logging-CRD is looking like follows:
+```yaml
+fluentd:
+  nodeSelector:
+    type: cpu
+  tolerations:
+  - effect: NoSchedule
+    key: type
+    operator: Equal
+    value: cpu
+```
+
+Additionaly we will have to increase the resources that are requested by the fluentd Pods. In the default setting they use following requests and limits:
+```yaml
+- Limits:
+  - cpu: 1000m
+  - memory: 400M
+- Requests:
+  - cpu: 500m
+  - memory:  100M
+```
+
+In this short walkthrough, we will increase the fluentd workers from `1` to `5`. Therefore, we will multiply the requests and limits with factor 5 to ensure enough resources are reserved. Additionally, we will set requests and limits to the same values to ensure that the fluentd Pods are not affected by other workloads on the Node. This is, in general, a good practice. We do this by changing the Logging-CRD like follows:
+```yaml
+fluentd:
+  nodeSelector:
+    type: cpu
+  tolerations:
+  - effect: NoSchedule
+    key: type
+    operator: Equal
+    value: cpu
+  resources:
+    limits:
+      cpu: 5
+      memory: 2G
+    requests:
+      cpu: 5
+      memory: 2G
+```
+
+The fluentd-Pods should receive their input and buffer them on their filesystem. After that, the workers can pick the logs up, process and forward them to their final destination. For this, we will have to configure a PVC and a buffer volume:
+
+```yaml
+fluentd:
+  nodeSelector:
+    type: cpu
+  tolerations:
+  - effect: NoSchedule
+    key: type
+    operator: Equal
+    value: cpu
+  resources:
+    limits:
+      cpu: 5
+      memory: 2G
+    requests:
+      cpu: 5
+      memory: 2G
+  rootDir: /buffers
```

Comment:
This setting should be changed together with the workers setting, I would move it there and highlight these should be set together.

Reply:
I will change that

Reply:
changed